pep-0458.txt

```diff
@@ -1183,6 +1183,45 @@ This PEP adds the ability for clients to use TUF metadata to improve the
 security of the update process.
 
 
+Hash Algorithm Transition Plan
+------------------------------
+
+If the algorithm used to hash target and metadata files becomes vulnerable, it
```

Comment:
```suggestion
If the algorithm used to hash target and metadata files becomes vulnerable to hash collision attacks, it
```
is it worth being explicit here? I'm not sure.

Reply:
When I wrote this I actually thought that it might be too explicit. Because the transition plan stays the same no matter the reason for changing the hash algorithm. :)

Reply:
I'm more than OK with that decision.